.gitlab/ci/.gitlab-ci.global.yml

```
@@ -225,21 +225,12 @@
 .checkout-upload-commit-content-nightly: &checkout-upload-commit-content-nightly
   - section_start "Checkout upload commit content nightly"
   - | 
-    if [[ -n "${NIGHTLY}" && "${CI_COMMIT_BRANCH}" == "master" ]]; then
-      echo "content nightly: checking out to the last successful upload commit $LAST_UPLOAD_COMMIT for the Packs folder"
+    if [[ "${CI_COMMIT_BRANCH}" == "fix_nightly_files_not_found" ]]; then
```

```suggestion
    if [[ -n "${NIGHTLY}" && "${CI_COMMIT_BRANCH}" == "master" ]]; then
```

after finishing up testing